aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Views/Shared/Layout/_Header.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿<nav class="main-header navbar navbar-expand navbar-white navbar-light">
+    @await Html.PartialAsync("_Header.LeftNavbar.cshtml")
+    @*@await Html.PartialAsync("_Header.SearchForm.cshtml")*@
```

Comment:
No i didn't forget that. There are also other commented code blocks too. [Here](https://github.com/iyilm4z/module-zero-core-template/blob/352a70cea875437b17ea66498df809f2dbbacd3a/aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Views/Shared/Layout/_Header.RightNavbar.cshtml#L5) are similar ones. I commented them out because users may want to implement them just by uncommenting them easily. I also would implement them later with a simple dashboard included. There would be a flag which makes theme on basic or advanced mode.

Reply:
It would be better to remove not used code blocks as much as possible. I mentioned before but probably you forgot. It would be better if we can first port the existing template to Admin LTE and than make such enhancements one by one in the future. Otherwise, it gets harder to review such PRs for us.

Reply:
So all i need to do is delete unneeded codes which my most favorite thing:) i'll do it with a new change set soon.

Reply:
Before you delete them you better store them in another branch if you think you will create pr later.

Reply:
Sure. I already did it. Thanks for your attention.

Reply:
@ismcagdas Previous theme had RightSideBar compnent to change theme color, in this new theme implementation we don't have that feature yet. However it's on my list of things to to soon. That's why there is no reason to have that component and it's app service&dtos, should i delete them too? Also I would name that new component which does this feature as ControlSidebar, because AdminLTE named it as it is.